sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Assessments"
+        :routeLocation="goBackRouteParams"
+        subTitle="View Request"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestDetails.status ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithSABC
+          "
+        >
+          <v-row class="p-0 m-0">
+            <v-btn color="primary" variant="outlined"
+              >Decline reassessment</v-btn
+            >
+            <v-btn class="ml-2" color="primary">Approve reassessment</v-btn>
+          </v-row>
+        </template>
+      </header-navigator>
```

Comment:
Regarding these actions, should there be a role associated with it?

Reply:
Role is part of next story.

Reply:
Yes, roles including any other button functionality is a part of the next story.